src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/services/OcrDataParser.java

```diff
@@ -44,8 +41,7 @@ public OcrDataParser(CcdCollectionParser ccdCollectionParser) {
                     )
                 );
             } catch (CcdDataParseException e) {
-                log.warn("Failed to parse OCR data", e);
-                return errors("Form OCR data has invalid format");
+                throw new CallbackProcessingException("Failed to parse OCR data from exception record", e);
```

Comment:
This is done by analogy to how it works for scanned documents.

https://github.com/hmcts/bulk-scan-ccd-event-handler-sample-app/pull/12 changes the name of the exception to more relevant.